SliderButton/SliderButtonBehaviorDecorator.js

```diff
@@ -10,43 +12,71 @@ import {findDOMNode} from 'react-dom';
  * @memberof agate/SliderButton
  * @private
  */
-const SliderButtonBehaviorDecorator = (Wrapped) => {
-	// eslint-disable-next-line no-shadow
-	function SliderButtonBehaviorDecorator ({onChange, ...rest}) {
-		const {children} = rest;
-		const [valueText, setValueText] = React.useState(children ? children[0] : null);
-		const ref = React.useRef();
-
-		function handleChange ({value}) {
-			onChange();
-			setValueText(children[value]);
+const SliderButtonBehaviorDecorator = hoc((config, Wrapped) => {
+	return class extends React.Component {
+		static displayName = 'SliderButtonBehaviorDecorator';
+
+		static propTypes = /** @lends agate/ArcSlider.ArcSliderBehaviorDecorator.prototype */ {
```

Comment:
It might not be updated after copy & paste.

```suggestion
		static propTypes = /** @lends agate/SliderButton. SliderButtonBehaviorDecorator.prototype */ {
```

Reply:
Uups. Thanks for noticing!